lib/merit/controller_extensions.rb

```diff
@@ -47,7 +47,7 @@ def had_errors?
     end
 
     def target_object
-      target_obj = instance_variable_get(:"@#{controller_name.singularize}")
+      target_obj = instance_variable_defined? ? instance_variable_get(:"@#{controller_name.singularize}") : nil
```

Comment:
@yakschuss we are not passing an argument to `instance_variable_defined?`! Suggested rewrite:
```ruby
def target_object
  variable_name = :"@#{controller_name.singularize}"
  if instance_variable_defined?(variable_name)
    target_obj = instance_variable_get(variable_name)
    if target_obj.nil?
      str = '[merit] No object found, you might need a ' \
        "'@#{controller_name.singularize}' variable in " \
        "'#{controller_path}_controller' if no reputation is applied. " \
        'If you are using `model_name` option in the rule this is ok.'
      Rails.logger.warn str
    end
  end
  target_obj
end
```